stac_fastapi/elasticsearch/stac_fastapi/elasticsearch/database_logic.py

```diff
@@ -79,7 +89,11 @@ def get_item_collection(
 
         # search = search.sort({"id.keyword" : {"order" : "asc"}})
         search = search.query()[0:limit]
-        collection_children = search.execute().to_dict()
+
+        body = search.to_dict()
```

Comment:
switched this from using the ES DSL method on search to generating the search and calling client.search, because we're going to convert this to be async next, and that doesn't work with the DSL execute and count methods.